packages/lib/src/components/internal/BaseElement/BaseElement.ts

```diff
@@ -178,6 +189,9 @@ abstract class BaseElement<P extends BaseElementProps> implements IBaseElement {
      * Unmounts a payment element from the DOM
      */
     public unmount(): this {
+        // Remove listener
+        off(this._node, 'keypress', this.handleKeyPress);
```

Comment:
Could the secured field remove all the event listeners when unmounting itself?

Reply:
This listener doesn't apply to securedFields - this is for UIElements within Checkout